openapi/platform/platform.yaml

```diff
@@ -1084,7 +763,7 @@ components:
         relationships:
           type: object
           properties:
-            businessPartner:
+            partner:
```

Comment:
`businessPartner` was selected for the field so that there is less confusion with vendor and fulfillment partners used by the business location. The collective group of all organizations the current user knows about are `partners`. 
It was done in alignment with the terms that the VBOC team choose to use for communications. I don't have strong opinions.

Reply:
Lets chat more about this one in the am. I feel we should just use partner and avoid the confusion of all of the different type possibilities.